lib/Minz/Extension.php

```diff
@@ -4,35 +4,26 @@
  * The extension base class.
  */
 abstract class Minz_Extension {
-	/** @var string */
-	private $name;
-	/** @var string */
-	private $entrypoint;
-	/** @var string */
-	private $path;
-	/** @var string */
-	private $author;
-	/** @var string */
-	private $description;
-	/** @var string */
-	private $version;
+	private string $name;
```

Comment:
This was on purpose that I did not update that in the dedicated PHP 7.4 PR https://github.com/FreshRSS/FreshRSS/pull/5720 because this might break inheritance for existing extensions. Have you tried?

Reply:
the inheritance is ok for me